### PR TITLE
[IMPROVEMENT] Clang format

### PR DESCRIPTION
**Pull request by NilsIrl**
**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

I am an active contributor to CCExtractor.

---

https://github.com/CCExtractor/ccextractor/pull/1219#issuecomment-580125561

requires #1219

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
@NilsIrl Can you rebase your branch against our current master to minimize the diff? :)

**Comment by NilsIrl** [comment]
> @NilsIrl Can you rebase your branch against our current master to minimize the diff? :)

Done

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Format
```
Comment (canihavesomecoffee) [inline .github/workflows/format.yml]:
```suggestion
name: Format sourcecode
```

**Comment by canihavesomecoffee** [comment]
@cfsmp3 Are you fine with the clang formatting rules?

**Pull request merged**